core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -144,27 +145,26 @@ protected String writeNewMetadata(TableMetadata metadata, int newVersion) {
     return newMetadataLocation.location();
   }
 
-  protected void refreshFromMetadataLocation(String newLocation) {
-    refreshFromMetadataLocation(newLocation, null, 20);
+  protected boolean refreshFromMetadataLocation(String newLocation, String previousLocation) {
+    return refreshFromMetadataLocation(newLocation, previousLocation, null, 20);
   }
 
-  protected void refreshFromMetadataLocation(String newLocation, int numRetries) {
-    refreshFromMetadataLocation(newLocation, null, numRetries);
+  protected boolean refreshFromMetadataLocation(String newLocation, String previousLocation, int numRetries) {
+    return refreshFromMetadataLocation(newLocation, previousLocation, null, numRetries);
   }
 
-  protected void refreshFromMetadataLocation(String newLocation, Predicate<Exception> shouldRetry,
-                                             int numRetries) {
+  protected boolean refreshFromMetadataLocation(
```

Comment:
iceberg style is to put the new line only before the arguments that don't fit within the line width. So

foo( x, y,
  z)

Not 
foo (x,
 y, 
 z)

Reply:
ok